src/intrinsics.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2009-2012, Salvatore Sanfilippo <antirez at gmail dot com>
+ * All rights reserved.
```

Comment:
this is not accurate and I don't think we include a license header for new files.

Reply:
Removed